src/connectors/infinite-hits/connectInfiniteHits.ts

```diff
@@ -147,13 +147,13 @@ const connectInfiniteHits: InfiniteHitsConnector = (
         };
 
         if (!escapeHTML) {
-          return parameters;
+          return new algoliasearchHelper.SearchParameters(parameters);
```

Comment:
Why don't we use the given value? It would avoid to use the constructor.

Reply:
setting any parameter calls the constructor anyway